code/game/objects/structures.dm

```diff
@@ -32,6 +32,7 @@
 		var/turf/T = get_turf(src)
 		spawn(0)
 			QUEUE_SMOOTH_NEIGHBORS(T)
```

Comment:
The fact this runs **after** the proc hurts. 

I am also sure this 100% contributes to GC fails because `spawn` """threads""" hold a ref to the caller.

Reply:
Smoothing neighbors is fine to run after, since that just smooths everything around it, not sure why it's behind a spawn(0) though

Reply:
works fine without the spawn, yeeted